site/diagrams/sequenceDiagrams/command_control_concept_http.txt

```diff
@@ -0,0 +1,18 @@
+participant Device
+participant HTTP Adapter
+participant AMQP 1.0 Network
```

Comment:
the network doesn't seem to be involved, so remove it

Reply:
The only thing is, that the business app connects to the network. But it adds nothing important to the sequence. But if I remove it, the reader might think the application connects directly to the adapter.